### PR TITLE
cleanup-and-rename

### PR DESCRIPTION
**Pull request by sertonix**
This pull request does the following:
- remove unused `.md` files
- removes badges from `README.md`
- updates repository url

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Atom API autocompletions",
```
Comment (confused-Techie) [inline package.json]:
```suggestion
  "description": "Pulsar API autocompletions",
```

Again might as well be thorough on these ones

**Review by confused-Techie — CHANGES_REQUESTED** [review]
Added one small change, but otherwise this looks great!

File: package.json (hunk 1 above)
Comment (sertonix) [reply package.json]: I feel like this could also be `API autocompletions` but I will do this change for now.

**Review by confused-Techie — APPROVED** [review]
This looks great to me!

**Pull request merged**